site/docs/aws.md

```diff
@@ -358,6 +361,16 @@ OPTIONS (
 PARTITIONED BY (category);
 ```
 
+We can then insert a single row into this new table
+```SQL
+INSERT INTO my_catalog.my_ns.my_table VALUES (1, "Pizza", "orders");
+```
+
+Which will cause the following object to be written to S3:
+```
+s3://my-table-data-bucket/2d3905f8/my_ns.db/my_table/category=orders/00000-0-5affc076-96a4-48f2-9cd2-d5efbc9f0c94-00001.parquet
+```
+
```

Comment:
add an explanation like "2d3905f8 is a hash generated based on the original folder-based path, so that all the file reads are evenly distributed across the S3 bucket"

Reply:
And "the shorter the shared S3 prefix is, the better performance we will get. So it is recommended to use a root bucket as your object storage path to achieve best performance."

Then we can talk about the path resolution strategy `write.object-storage.path` -> `write.folder-storage.path` -> table's default root location + `/data`

Reply:
added